workloads/network-perf/common.sh

```diff
@@ -11,6 +11,7 @@ export_defaults() {
   network_type=$(oc get network cluster -o jsonpath='{.status.networkType}' | tr '[:upper:]' '[:lower:]')
   export UUID=$(uuidgen)
   export CLUSTER_NAME=$(oc get infrastructure cluster -o jsonpath="{.status.infrastructureName}")
+  export PROM_TOKEN=$(oc sa get-token -n openshift-monitoring prometheus-k8s || oc sa new-token -n openshift-monitoring prometheus-k8s)
```

Comment:
Ran some of this manually and saw : 
`Command "new-token" is deprecated, and will be removed in the future version. Use oc create token instead.`

Reply:
looks like a 1.24/4.11 change, updated
Thanks